tools/packaging/guest-image/build_image.sh

```diff
@@ -59,11 +59,11 @@ build_image() {
 	info "Build image"
 	info "image os: $os_name"
 	info "image os version: $os_version"
-	make image \
+	make invalidate-rootfs image \
```

Comment:
I like the new `invalidate-rootfs` target but I'm not sure we should do this: this means we're changing the behavior of `make rootfs-image-tarball` and it's not idempotent anymore. Curious to hear what others think.

Reply:
Ok. I've addressed this with "artifact-version" saved in rootfs mark file.